crates/sovereign-sdk/rollup-interface/src/state_machine/zk/mod.rs

```diff
@@ -26,8 +26,9 @@ use crate::spec::SpecId;
 /// The ZK proof generated by the [`ZkvmHost::run`] method.
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, BorshSerialize, BorshDeserialize)]
 pub enum Proof {
+    /// Mock proof
     /// Only public input was generated.
```

Comment:
remove all of this public input stuff, now we have the receipt for execute mode so the output has to be same